3.processing_features/scripts/0.merge_sc_cytotable.py

```diff
@@ -59,52 +75,27 @@
                 """
 
 
+# ## Create dictionary with info for each plate
+# 
+# **Note:** All paths must be string to use with CytoTable.
+
 # In[3]:
 
 
-# dictionary with info for the sqlite file from each plate
+# create plate info dictionary with all parts of the CellProfiler CLI command to run in parallel
 plate_info_dictionary = {
-    "Plate_1": {
-        # path to outputted SQLite file
-        "source_path": str(
-            pathlib.Path(
-                "../2.cellprofiler_analysis/analysis_output/Plate_1.sqlite"
-            )
-        ),
-        "dest_path": str(pathlib.Path(f"./data/converted_data/Plate_1.parquet")),
-    },
-    "Plate_2": {
-        # path to outputted SQLite file
-        "source_path": str(
-            pathlib.Path(
-                "../2.cellprofiler_analysis/analysis_output/Plate_2.sqlite"
-            )
-        ),
-        # path for merged single cell paraquet file (without annotations)
-        "dest_path": str(pathlib.Path(f"./data/converted_data/Plate_2.parquet")),
-    },
-    "Plate_3": {
-        # path to outputted SQLite file
-        "source_path": str(
-            pathlib.Path(
-                "../2.cellprofiler_analysis/analysis_output/Plate_3.sqlite"
-            )
-        ),
-        # path for merged single cell paraquet file (without annotations)
-        "dest_path": str(pathlib.Path(f"./data/converted_data/Plate_3.parquet")),
-    },
-    "Plate_3_prime": {
-        # path to outputted SQLite file
-        "source_path": str(
-            pathlib.Path(
-                "../2.cellprofiler_analysis/analysis_output/Plate_3_prime.sqlite"
-            )
-        ),
-        # path for merged single cell paraquet file (without annotations)
-        "dest_path": str(pathlib.Path(f"./data/converted_data/Plate_3_prime.parquet")),
+    name: {
```

Comment:
This is cleaner, good job

Reply:
May also consider including this in the for loop below

Reply:
Thank you! I will be adding here the same reply as the last suggestion regarding readability and easier interpretation of code by keeping the for loops separate.